src/lib/mac/kmac/kmac.cpp

```diff
@@ -0,0 +1,183 @@
+/*
+* KMAC
+* (C) 2023 Falko Strenzke
+*
+* Botan is released under the Simplified BSD License (see license.txt)
+*/
+
+#include "botan/assert.h"
+#include "botan/exceptn.h"
+#include "botan/secmem.h"
```

Comment:
This fixes the amalgamation CI builds:
```suggestion
#include <botan/assert.h>
#include <botan/exceptn.h>
#include <botan/secmem.h>
```

Reply:
Thanks for these corrections.